src/steamship/utils/kv_store.py

```diff
@@ -90,8 +92,24 @@ def set(self, key: str, value: Dict[str, Any]):
         req = Tag.CreateRequest(file_id=file.id, kind=self.store_identifier, name=key, value=value)
         return self.client.post("tag/create", req, expect=Tag)
 
+    def list(self, filter_keys: Optional[List[str]] = None) -> List[Tuple[str, Dict[str, Any]]]:
```

Comment:
I would call it .items() similar to how you would call dict.items()

Reply:
```suggestion
    def list(self, filter_keys: Optional[List[str]] = None) -> List[Tuple[str, Any]]:
```

I guess we also support strings, bools, ints, and floats as values. If yes, then we cover `Any`.

Reply:
At the moment it's hard-coded to only support `Dict` objects so the `Dict[str, Any]` is actually the correct type here unless we relax that requirement

Reply:
Updated to `items`